pkg/sfu/forwardstats.go

```diff
@@ -46,9 +48,17 @@ func (s *ForwardStats) Update(arrival, left time.Time) {
 
 func (s *ForwardStats) GetStats() (latency, jitter time.Duration) {
 	s.lock.Lock()
-	defer s.lock.Unlock()
 	w := s.latency.Summarize()
-	return time.Duration(w.Mean()), time.Duration(w.StdDev())
+	s.lock.Unlock()
+	latency, jitter = time.Duration(w.Mean()), time.Duration(w.StdDev())
+	// TODO: remove this check after debugging unexpected jitter issue
+	if jitter > 10*time.Second {
+		logger.Infow("unexpected forward jitter",
+			"jitter", jitter,
+			"stats", fmt.Sprintf("count %.2f, mean %.2f, stdDev %.2f", w.Count(), w.Mean(), w.StdDev()),
+		)
```

Comment:
is this possibly due to out-of-order packets?

Reply:
It calculates the stdDev of forward latency so there is no out-of-order packets here (it is a global stats for sfu node), and the value is very strange 2*10^9 milliseconds, confusing.

Reply:
ah, global number, stupid question from me 🤦‍♂️